docs/changes/3.0.0.rst

```diff
@@ -14,6 +14,105 @@ Changes to Supported Platforms
 
 .. towncrier release notes start
 
+
+
+
+Breaking Changes
+----------------
+
+- Starting with Falcon 3.0, Python 2 is no longer supported. Therefore, support
+  for CPython 2.7 and PyPy2.7 was removed in this release. We will continue to
+  maintain the Falcon 2.x series for those who still need to run under Python 2,
+  at least until that platform's scheduled EOL (January 1, 2020 at the time of
+  this writing; see also PEP 373). (27-support)
+- Python 3.4 has now reached its end-of-life per PEP 429, and as a result is no
+  longer supported as of Falcon 3.0. (34-support)
```

Comment:
I can't comment on a file that as just been renamed, so commenting here: these breaking changes seem like they duplicate what is already documented in "Changes to Supported Platforms". Do we still want them?

Reply:
No, I think we do not need to duplicate the "Changes to Supported Platforms" section.

Reply:
✅